charts/renku/values.yaml

```diff
@@ -575,13 +582,14 @@ notebooks:
     # rbac:
     #   enabled: true
     ## Set the notebooks service API token
-    # hub:
-    #   services:
+    hub:
```

Comment:
why is this uncommented? the gateway setting should be applied in the default values for the notebooks service.

Reply:
Because I removed this from the default values in the notebooks service for the secondary `renku-notebooks` deployment.

Reply:
IMO we shouldn't set any defaults for the various services here if at all possible - the values are provided here just as documentation.

Reply:
Why? Renku uses a dependency (renku-notebooks) in a specific way. The same dependency is used differently in other places. I think this is the natural place to put such settings.

Reply:
yes but that dependency is part of renku :) So if we want a default set for all to use we should just set it in the dependency itself.

Reply:
Agree, but now we have a use case where a child chart is used independently of the renku chart with different settings and then it starts to be more like a regular dependency. I thought this was the cleaner way to avoid bloat in the values file for the secondary notebooks deployment, but if you have a very strong opinion about this I think setting the `gateway` key to `Null` in the in the deployment values file for the secondary notebooks deployment should do the job too.